src/main/java/teammates/ui/webapi/CreateReadNotificationAction.java

```diff
@@ -0,0 +1,54 @@
+package teammates.ui.webapi;
+
+import java.time.Instant;
+
+import org.apache.http.HttpStatus;
+
+import teammates.common.datatransfer.attributes.AccountAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Logger;
+import teammates.ui.output.AccountData;
+import teammates.ui.request.InvalidHttpRequestBodyException;
+import teammates.ui.request.ReadNotificationCreateRequest;
+
+/**
+ * Action: Creates a read notification in account entity.
+ */
+public class CreateReadNotificationAction extends Action {
```

Comment:
Similar to `ReadNotificationCreateRequest`, can consider renaming to `MarkNotificationAsReadAction`.